code/__HELPERS/names.dm

```diff
@@ -1,8 +1,10 @@
+/* BUBBERSTATION CHANGE: REMOVED IN FAVOR OF ashwalker_names.dm
```

Comment:
I don't think you need to comment this out...

Reply:
you do because you can't override global procs

Reply:
> you do because you can't override global procs

yes you can just do

/myproc(args)

Reply:
ye just confirmed it